Lib/test/test_pprint.py

```diff
@@ -447,6 +479,28 @@ def test_mapping_proxy(self):
                           ('lazy', 7),
                           ('dog', 8)]))""")
 
+    def test_dict_views(self):
+        for dict_class in (dict, collections.OrderedDict):
+            empty = dict_class()
+            short = dict_class(zip('abcde', 'abcde'))
+            long = dict_class((chr(x), chr(x)) for x in range(65, 91))
+            prefix = "dict" if dict_class is dict else "odict"
+            for d in empty, short, long:
```

Comment:
This could use `subTest` (see unittest doc)

Reply:
subTest added, improves the testing a lot, thank you!